code/data-pipeline/tests/test_load_data.py

```diff
@@ -15,3 +17,52 @@ def test_load_data_sources():
     
     df = fdl.load_data()
     assert set(df['source'].unique()) == {'flusurvnet', 'hhs', 'ilinet'}
+
+
+def test_load_data_kwargs():
```

Comment:
Love to see the increasing test coverage with each PR!

At the end of the day, the coverage is what matters, but from a readability and smoother debugging perspective, this is a good candidate for more specific, smaller test functions (e.g., test_load_data_hhs_kwargs, test_load_data_ilinet_kwargs, etc.)

Reply:
Splitting this up would also enable the use of Pytest's parameterization feature. e.g.,:

```
@pytest.mark.parametrize("test_kwargs,season_expected,wk_end_date_expected", [
    ({}, '2022/23', '2023-12-23'),
    ({'drop_pandemic_seasons': False}, '2019/20', '2023-12-23'),
    ({'drop_pandemic_seasons': True, 'as_of': datetime.date.fromisoformat('2023-12-30')}, '2022/23', '2023-12-23')
])
def test_load_data_hhs_kwargs(test_kwargs, season_expected, wk_end_date_expected):
    fdl = FluDataLoader('../../data-raw')
    
    df = fdl.load_data(sources=['hhs'], hhs_kwargs=test_kwargs)
    wk_end_date_actual = str(df['wk_end_date'].max())[:10]
    assert df['season'].min() == season_expected
    if 'as_of' in test_kwargs:
        assert wk_end_date_actual == wk_end_date_expected
    else:
        assert wk_end_date_actual > wk_end_date_expected
```

Pytest treats each iteration of the parameters as a separate test, so the running the above (`pytest -k test_load_data_hhs_kwargs`) outputs 3 successful test runs

Reply:
Thanks for these suggestions! I went ahead with this and I'm a fan of the result.